packages/phrases/src/locales/pt-pt/translation/admin-console/sign-in-exp.ts

```diff
@@ -0,0 +1,116 @@
+const sign_in_exp = {
+  title: 'Experiência de login',
```

Comment:
We prefer using the term `sign in` rather than `login` in English. I'm no Portuguese expert, but can you use something similar to `sign in` in portuguese?

Reply:
lol. OK

Reply:
Hmmm, I don't know. Like I said, I'm no expert in Portuguese. You can decide! If you think `login` is more authentic, then we can also use `login`.

Reply:
You decide then, as long as it is semantically smooth and authentic

Reply:
OK, let's go with `login` for now